### PR TITLE
Updates the org drop down so it loads all orgs

### PR DESCRIPTION
**Pull request by MauriceReeves-usds**
This PR updates Docker to include the download site, and then updates the org drop down to include all the organizations if you're a Prime Admin

## Changes
- Updates the Docker file to unite the rest of the application and the static site
- Updates the logic in the dropdown to include all the organizations in the application
- Cleans up how we invoke axios for calls to the API

### Testing
- [x] Tested locally?
- [ ] Ran `quickTest all`?
- [ ] Ran `./prime test` against local Docker ReportStream container?
- [ ] Downloaded a file from `http://localhost:7071/api/download`?
- [ ] Added tests?

### Specific Security-related subjects a reviewer should pay specific attention to

- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Fixes
*List GitHub issues this PR fixes*
- #1459

### CODE REVIEW

**Review by Bisonburger — APPROVED** [review]
LGTM

File: frontend/src/js/prime-web-receiver-pre.js [hunk 1]
```diff
@@ -172,17 +184,27 @@ function logout() {
  *
  */
 async function fetchReports() {
-    const baseURL = getBaseUrl();
-    return isLocalhost()? ReportData : window.jwt? axios.get(`${baseURL}/api/history/report`, apiConfig()).then(res => res.data) : [];
+    let url = 'history/report';
```
Comment (td-usds) [inline frontend/src/js/prime-web-receiver-pre.js:187]: _Consider_ lifting this up as a constant; it's used in 3-4 different places.

File: frontend/src/js/prime-web-receiver-pre.js [hunk 2]
```diff
@@ -225,53 +275,37 @@ function processJwtToken(){
...
-            let _orgsOptions = "";
-
-            window.orgs.forEach( org => {
-                _orgsOptions +=
-                    `
-                        <option value="${org}" ${window.org === org ? 'selected="selected"' : ""} >
-                            ${convertOrgName(org).toUpperCase()}
-                        </option>
-                    `;
+        if (claims.organization.includes("DHPrimeAdmins")) {
```
Comment (td-usds) [inline frontend/src/js/prime-web-receiver-pre.js:296]: Consider making `DHPrimeAdmins` a constant; it's used here and on L305, L288 and probably some other places too

File: prime-router/src/main/kotlin/azure/HistoryFunctions.kt [hunk 3]
```diff
@@ -175,13 +176,31 @@ class GetReports :
     fun run(
         @HttpTrigger(
             name = "getReports",
-            methods = [HttpMethod.GET],
+            methods = [HttpMethod.GET, HttpMethod.HEAD, HttpMethod.OPTIONS],
```
Comment (td-usds) [inline prime-router/src/main/kotlin/azure/HistoryFunctions.kt:179]: Why the addition of `HEAD` and `OPTIONS even though `getReports` doesn't differentiate between these methods?

File: prime-router/src/main/kotlin/azure/HistoryFunctions.kt [hunk 4]
```diff
@@ -175,13 +176,31 @@ class GetReports :
...
+}
+
+class GetReportsByOrganization :
+    BaseHistoryFunction() {
+    @FunctionName("getReportsByOrganization")
+    @StorageAccount("AzureWebJobsStorage")
+    fun run(
+        @HttpTrigger(
+            name = "getReportsByOrganization",
+            methods = [HttpMethod.GET, HttpMethod.HEAD, HttpMethod.OPTIONS],
```
Comment (td-usds) [inline prime-router/src/main/kotlin/azure/HistoryFunctions.kt:196]: See L179, why also `HEAD` and `OPTIONS`?

File: prime-router/docker-compose.yml [hunk 5]
```diff
@@ -109,6 +109,9 @@ services:
       - prime_dev
     command: "'nginx' '-g' 'daemon off;'"
     entrypoint: "/docker-entrypoint.sh"
+    build:
+      context: ../frontend
+      dockerfile: ../frontend/Dockerfile_dev
```
Comment (td-usds) [inline prime-router/docker-compose.yml:114]: Nitpick: Prefer calling that file `Dockerfile.dev` (separator being a `.`. instead of an `_`)

**Review by td-usds — CHANGES_REQUESTED** [review]

**Pull request merged**